src/popup/components/TransferReview.vue

```diff
@@ -389,13 +398,22 @@ export default defineComponent({
       isSelectedAssetAex9,
       tokenSymbol,
       account,
+      activeMultisigAccount,
     };
   },
 });
 </script>
 
 <style scoped lang="scss">
-.details-item {
-  margin-top: 16px;
+.review-wrapper {
```

Comment:
I suggest to rename the root class of the component to it's name, but in kebab-case as in every other place in the project.